src/corbado_python_sdk/exceptions/token_validation_exception.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum
+
+from typing_extensions import Optional
+
+
+class ValidationErrorType(Enum):
+    """
+    Enum representing types of validation errors.
+
+    This enum categorizes various validation errors that may occur during
+    token validation processes.
+
+    Attributes:
+        INVALID_TOKEN (str): Indicates that the token is invalid. More information in 'original_exception'.
+        SIGNING_KEY_ERROR (str): Indicates that the signing key could not be retrieved. More information in 'original_exception'.
+        EMPTY_SESSION_TOKEN (str): Indicates that the session token is empty.
+        EMPTY_ISSUER (str): Indicates that the issuer is empty.
+        ISSUER_MISSMATCH (str): Indicates that the token issuer does not match the expected issuer.
+    """
+
+    INVALID_TOKEN = "Invalid token"  # noqa s105
```

Comment:
shouldnot we have the following codes like in go:

CodeJWTGeneral Code = iota
	CodeJWTIssuerMismatch
	CodeJWTInvalidData
	CodeJWTInvalidSignature
	CodeJWTBefore
	CodeJWTExpired
	CodeJWTIssuerEmpty

Reply:
Done, but due to different JWT implementation in Java and Python and error handling not all abovementioned error may get caught.